src/types/main.ts

```diff
@@ -58,19 +56,12 @@ export interface LinkItem {
   link: string;
 }
 
-export interface BuildingFloor {
+export interface FloorOverlay {
```

Comment:
Again, changes from #58 have been reverted

Reply:
Seems I've made a mistake confusing BuildingFloor as being the older type as such reverted #58  , fixing it now. Thanks for the catch 😅